src/commands/publish.ts

```diff
@@ -21,22 +22,39 @@ import { Ignore } from "../context/ignore.ts";
 
 import { getAPIKey } from "../keyfile.ts";
 import { version } from "../version.ts";
+import { setupLog } from "../log.ts";
+
+interface File {
+  fullPath: string;
+  path: string;
+  lstat: Deno.FileInfo;
+}
+
+const nullConfig = {
+  name: "",
+  files: [""],
+};
+
+const nullIgnore = {
+  accepts: [],
+  denies: [],
+};
 
 async function getContext(): Promise<[Config, Ignore]> {
```

Comment:
... consider returning `undefined` instead.

```suggestion
async function getContext(): Promise<[Config | undefined, Ignore | undefined]> {
```

Reply:
I'm gonna do this 😉